go.mod

```diff
@@ -135,3 +135,6 @@ require (
 )
 
 replace github.com/aws/amazon-sagemaker-operator-for-k8s => github.com/aws/amazon-sagemaker-operator-for-k8s v1.0.1-0.20210303003444-0fb33b1fd49d
+
+// TODO: remove this when flyteidl is ready
```

Comment:
@honnix Remove this after https://github.com/flyteorg/flyteidl/pull/422 is merged.